src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -14,9 +14,14 @@
 using Microsoft.Azure.Templates.Analyzer.Reports;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json.Linq;
+using Analyzer.Cli;
```

Comment:
After the namespace change in ExitCode.cs, this will no longer be needed.

Reply:
Good catch!